derp/derp_server.go

```diff
@@ -47,6 +48,20 @@ import (
 
 var debug, _ = strconv.ParseBool(os.Getenv("DERP_DEBUG_LOGS"))
 
+// serverTotalAllowanceKB is the number of allowed KB to be transferred across this DERP server
```

Comment:
Is this a total transfer limit or a bandwidth RATE limit? why is this in terms of KB and not in terms of bytes? Why not 1<<40?

Reply:
I was thinking that bytes would be too granular, maybe just eager optimization on my part.

Reply:
1 << 40 overflows int on 32 bit architectures, which might be one argument for KB

Reply:
Don't use an int then. :)

See:
https://blog.golang.org/constants 
https://tour.golang.org/basics/16

https://golang.org/ref/spec#Constants ...

> An untyped constant has a default type which is the type to which the constant is implicitly converted in contexts where a typed value is required, for instance, in a short variable declaration such as i := 0 where there is no explicit type. The default type of an untyped constant is bool, rune, int, float64, complex128 or string respectively, depending on whether it is a boolean, rune, integer, floating-point, complex, or string constant.